src/client/src/apps/MainRoute/widgets/spotTile/components/stories/PriveMovement.stories.tsx

```diff
@@ -6,13 +6,15 @@ import { select } from '@storybook/addon-knobs'
 stories.add('Price movement', () => {
   const priceMovementType = select('Direction', { Up: 'Up', Down: 'Down' }, 'Up')
   const showPriceMovement = true
+  const isAnalyticView = true
```

Comment:
Why not change this to a `boolean` story knob?
```suggestion
  const isAnalyticView = boolean("Analytic View", true)
```

Reply:
good point, I will look into storybook and especially for story knobs